Mage/src/main/java/mage/abilities/effects/common/DrawCardSourceControllerEffect.java

```diff
@@ -28,10 +30,18 @@ public DrawCardSourceControllerEffect(DynamicValue amount) {
         this(amount, false);
     }
 
+    public DrawCardSourceControllerEffect(DynamicValue amount, ValuePhrasing textPhrasing) {
```

Comment:
For info: I don’t know why devs ignore rules samples in text gen code but it much more easy to read and research with examples like that and/or real card refs:

![IMG_0743](https://github.com/user-attachments/assets/ba84865d-7cc5-4723-8a9e-421908a06270)